config/routes.php

```diff
@@ -1,6 +1,6 @@
 <?php
 use Cake\Routing\Router;
 
-Router::plugin('Burzum/FileStorage', function($routes) {
-	$routes->fallbacks();
-});
\ No newline at end of file
+$routes->plugin('Burzum/FileStorage', function($routes) {
```

Comment:
A doc block with a type hint would be nice to have.

```suggestion
/** @var \Cake\Routing\RouteBuilder $routes */
$routes->plugin('Burzum/FileStorage', function($routes) {
```